backend-server/src/api/controllers/eventController.ts

```diff
@@ -0,0 +1,55 @@
+import { Timestamp } from "@firebase/firestore";
+import { createHash } from "../../util/createHash";
+import { dataAccess } from "../data/dataAccess";
+import { Event } from "../data/models";
+
+const get_uri: string = "localhost:8080/v1/event/";
+
+export async function addEvent(
+  organization: string,
```

Comment:
Let's create an `AddEventBody` model and use that instead of passing around all these values individually. See `availabilityController.ts` for an example of the controller

Reply:
That way we can also validate the request body is valid at the router layer. (See `availability.ts` for an example of the router)